electrum/tests/test_auxpow.py

```diff
@@ -37,61 +69,52 @@ def test_deserialize_auxpow_header_explicit_coinbase(self):
         self.assertEqual(expected_coinbase_txid, observed_coinbase_txid)
 
         coinbase_merkle_branch = header_auxpow['coinbase_merkle_branch']
```

Comment:
It looks like this PR removes all code that uses the `coinbase_merkle_branch` variable.  If that's accurate, I think this variable should be removed as well since it's unused.

Reply:
Indeed, good catch!  I've removed the unused variable.